frontera/contrib/backends/sqlalchemy/components.py

```diff
@@ -173,10 +173,10 @@ def get_next_requests(self, max_n_requests, partition_id, **kwargs):
         try:
             for item in self._order_by(self.session.query(self.queue_model).filter_by(partition_id=partition_id)).\
                     limit(max_n_requests):
-                method = item.method or 'GET'
+                method = item.method or b'GET'
```

Comment:
The type here doesn't matter because in the next line we initialize the Request object with method and it is converted to bytes in the Request model.